new/LIT7336QeneMaryam.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7336QeneMaryam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ማርያም፡ ደከምኪ፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Māryām dakamki (Qǝne of the mawaddǝs kʷǝllǝkkǝmu-type)</title>
+                <title xml:lang="en" corresp="#t1">Mary, you became exhausted ...</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne of the mawaddǝs-type as indicated in the edition of <bibl><ptr target="bm:Afawarq1984Yageezqeneyat"/>
+                    <citedRange unit="page">307</citedRange></bibl>, having 9 verses resulting in a mawaddǝs kʷǝllǝkkǝmu-type according to
+                    the classification of <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>. The 
+                    author is not indicated, neither in the aforementioned edition, nor in the manuscript witness in 
```

Comment:
I think it would be good to indicate that only two lines have been preserved in Abb 145, due to whatever reasons (maybe just a mistake of the writer).